h2o-bindings/bin/custom/python/gen_infogram.py

```diff
@@ -0,0 +1,323 @@
+else: # it is set
+    if self._parms["protected_columns"] is not None:    # fair infogram
+        self._parms["relevance_index_threshold"] = relevance_index_threshold
+    else: # core infogram should not have been set
+        warnings.warn("Should not set relevance_index_threshold for core infogram runs.  Set total_information_threshold instead.  Using default of 0.1 if not set", RuntimeWarning)
+"""
+    ),
+    safety_index_threshold=dict(
+        setter="""
+if safety_index_threshold <= 4.940656458e-324: # not set
+    if self._parms["protected_columns"] is not None:
+        self._parms["safety_index_threshold"]=0.1
+else: # it is set
+    if self._parms["protected_columns"] is not None: # fair infogram
+        self._parms["safety_index_threshold"] = safety_index_threshold
+    else: # core infogram should not have been set
+        warnings.warn("Should not set safety_index_threshold for core infogram runs.  Set net_information_threshold instead.  Using default of 0.1 if not set", RuntimeWarning)
+"""
+    ),
+    net_information_threshold=dict(
+        setter="""
+if net_information_threshold <= 4.940656458e-324: # not set
+    if self._parms["protected_columns"] is None:
+        self._parms["net_information_threshold"]=0.1
+else:  # set
+    if self._parms["protected_columns"] is not None: # fair infogram
+        warnings.warn("Should not set net_information_threshold for fair infogram runs.  Set safety_index_threshold instead.  Using default of 0.1 if not set", RuntimeWarning)
+    else:
+        self._parms["net_information_threshold"]=net_information_threshold
+"""
+    ),
+    total_information_threshold=dict(
+        setter="""
+if total_information_threshold <= 4.940656458e-324: # not set
+    if self._parms["protected_columns"] is None:
+        self._parms["total_information_threshold"] = 0.1
+else:
+    if self._parms["protected_columns"] is not None: # fair infogram
+        warnings.warn("Should not set total_information_threshold for fair infogram runs.  Set relevance_index_threshold instead.  Using default of 0.1 if not set", RuntimeWarning)
+    else:
+        self._parms["total_information_threshold"] = total_information_threshold
+"""
+    )
+)
+
+doc = dict(
+    __class__="""
+Given a sensitive/unfair predictors list, Infogram will add all predictors that contains information on the 
+ sensitive/unfair predictors list to the sensitive/unfair predictors list.  It will return a set of predictors that
+ do not contain information on the sensitive/unfair list and hence user can build a fair model.  If no sensitive/unfair
+ predictor list is given, Infogram will return a list of core predictors that should be used to build a final model.
+ Infogram can significantly cut down the number of predictors needed to build a model and hence will build a simple
+ model that is more interpretable, less susceptible to overfitting, runs faster while providing similar accuracy
+ as models built using all attributes.
```

Comment:
Second to last line have unnecessary indentation (could cause some formatting issue in generated docs).

Reply:
Fixed.